reports-scheduler/src/test/kotlin/org/opensearch/reportsscheduler/TestHelpers.kt

```diff
@@ -54,21 +54,13 @@ fun constructReportDefinitionRequest(
                         "origin":"localhost:5601",
                         "id":"id"
                     },
+                    $trigger
                     "format":{
                         "duration":"PT1H",
                         "fileFormat":"Pdf",
                         "limit":1000,
                         "header":"optional header",
                         "footer":"optional footer"
-                    },
-                    $trigger
```

Comment:
How come we are removing the entire section? If `recipients` and `deliveryFormat` fields are removed, does the CI still fail? Those 2 fields are not present in the new delivery schema

Reply:
the CI fails as you commented before because it sees delivery parameter and calls notifications plugin, which is not installed. I'm working on bundling notifications into integTest but currently seeing some issues, meanwhile removing the parameter temporarily to unblock CI since I need github action artifacts to test something

Reply:
Got it 👍